tests/handlers/external_test.py

```diff
@@ -11,14 +11,15 @@
 from jinja2 import Environment, PackageLoader, select_autoescape
 from lsst.daf.butler import LabeledButlerFactory
 
-from datalinker.config import config
+from datalinker.dependencies.config import config_dependency
 
 from ..support.butler import MockButler
 
 
 @pytest.mark.asyncio
 async def test_get_index(client: AsyncClient) -> None:
     """Test ``GET /api/datalink/``."""
+    config = config_dependency.config()
```

Comment:
I would drop this from the tests that don't use config. I'd also be tempted to make a config fixture that sets up the right config path and returns the loaded test configuration, so `config` can be a test function argument instead.